platform/linuxbsd/x11/dynwrappers/xcursor-so_wrap.c

```diff
@@ -1,12 +1,8 @@
 // This file is generated. Do not edit!
 // see https://github.com/hpvb/dynload-wrapper for details
-// generated by generate-wrapper.py 0.3 on 2023-01-23 15:09:53
-// flags: generate-wrapper.py --include ./thirdparty/linuxbsd_headers/X11/Xcursor/Xcursor.h --sys-include "thirdparty/linuxbsd_headers/X11/Xcursor/Xcursor.h" --soname libXcursor.so.1 --init-name xcursor --output-header ./platform/linuxbsd/x11/dynwrappers/xcursor-so_wrap.h --output-implementation ./platform/linuxbsd/x11/dynwrappers/xcursor-so_wrap.c
+// generated by /home/hp/Projects/dynload-wrapper/generate-wrapper.py 0.6 on 2024-11-08 22:42:37
```

Comment:
We might have been editing the cwd away previously to avoid unnecessarily including personal info in the logs.

Might be worth still doing, and maybe tweaking the script to only write the filename.

Reply:
Oh yeah, I can update the script to only output the basename. I'll force push to this PR.

Reply:
fixed in https://github.com/hpvb/dynload-wrapper/commit/b48ca6032bfd2dbdea6e0390ed9e17e90c2d5ccb